### DISCUSSION TITLE
Good to have grouping of file extension in the filter webpart

### DISCUSSION THREAD

**Discussion by sumitdeyonline** [microsoft-search/pnp-modern-search#1598] [Ideas]
**Is your feature request related to a problem? Please describe.**
No, Nice to have

**Describe the solution you'd like**
Please provide a groping of file extension in the filter webpart, suppose we have files with the different image extension, like .jpg, .png, .gif, .jpeg, if it is possible to group these file extension in the filter with a single word, like below

- [ ] Image (.jpg, .png, .gif, .jpeg, etc.) [That can be configured by the user]

- [ ] Word Document (.doc, docx)

- [ ] HTML(.html, .htm etc.)

When user click on the Image checkbox all image file would show up in the search result. 

**Describe alternatives you've considered**
There are 100 of extensions of our search result, it very different browse through the list.  

**Additional context**
It is available on the Microsoft classic search.

**Comment by wobba** [comment]
You can do this with a custom filter template in v4 as a workaround.

**Comment by sumitdeyonline** [comment]
Thank you for your suggestion, do you have any syntax documentation on
custom filter templates.

**Reply by wobba** [reply]
https://microsoft-search.github.io/pnp-modern-search/usage/search-filters/layouts/#custom
